lib/cart.rb

```diff
@@ -0,0 +1,59 @@
+require_relative "menu_item"
+require_relative "menu"
+require_relative "order_item"
+require_relative "customer"
+
+class Cart
+  attr_accessor :cart
+  attr_accessor :total_cost
+
+  def initialize
+    @cart = []
+    @total_cost = 0
+  end
+
+  @@instance = Cart.new
+
+  def self.instance
+    return @@instance
+  end
+
+  def add_item_to_cart(order_item)
+    @cart.push(order_item)
+    cost = order_item.cost.to_i
+    @total_cost += cost
+  end
+
+  def to_s
```

Comment:
Ok, thanks.  Regarding the `add_item_to_cart` method, I am thinking I will move `@cart.push(order_item)` (line 22) so it is the last thing done in the method so that the `cart` will be returned by the method.  Not sure if that matters in this case, but I wanted to write this comment to articulate the thought in hopes of better understanding it.